velox/expression/tests/SimpleFunctionTest.cpp

```diff
@@ -372,7 +372,8 @@ struct ArrayRowReaderFunction {
       const arg_type<Array<Row<int64_t, double>>>& input) {
     out = 0;
     for (size_t i = 0; i < input.size(); i++) {
-      out += *input.at(i)->template at<0>();
+      auto&& row = *input.at(i);
+      out += *row.template at<0>();
```

Comment:
why not on one line?
```suggestion
      out += *input.at(i)->template at<0>();
```

Reply:
in one line it would be  (two stars) 
which is a bit overloaded.
```
      out += *((*input.at(i))->template at<0>());
```